web/admin/src/App.js

```diff
@@ -30,7 +31,10 @@ export default class App extends PureComponent {
       searchContent: [],
       search: false,
       userContent : [],
-      hidden : true
+      hidden : true,
+      openVar: false,
```

Comment:
We know it's a variable, so let's drop the `Var` and call it `showModal`, since, in the context of the whole App component, `openVar` doesn't tell us that it is related to the modal.